src/components/Audiences/index.js

```diff
@@ -15,7 +15,15 @@ import injectProcessDestinations from "./injectProcessDestinations";
 import injectProcessResponse from "./injectProcessResponse";
 
 const createAudiences = ({ logger, fireReferrerHideableImage }) => {
-  const loggingCookieJar = createLoggingCookieJar({ logger, cookieJar });
+  const cookieJarWithEncoding = cookieJar.withConverter({
```

Comment:
Can we please add a comment explaining why we are adding a converter?

Reply:
sure